phase1/gce/do

```diff
@@ -53,9 +61,17 @@ fetch_kubeconfig() {
     # succeeds, then stdout is tainted by ssh initialization output, so we echo
     # a marker to indicate the start of the file, and delete all lines up to and
     # including it.
-    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > ${TMP_DIR}/kubeconfig.raw; then
+    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$SSHUSER$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > ${TMP_DIR}/kubeconfig.raw; then
       sed '/^STARTFILE$/,$!d;/^STARTFILE$/d' ${TMP_DIR}/kubeconfig.raw > ${CLUSTER_NAME}/kubeconfig.json
       echo Successfully fetched kubeconfig.
+
+      # kubeconfig.json fetched from master node has the user, cluster and context all hardcoded with
+      # 'kubernetes' keyword. So replace 'kubernetes' with the cluster_name.
+      sed -i "s/kubernetes/${CLUSTER_NAME}/g" ${CLUSTER_NAME}/kubeconfig.json
```

Comment:
yeah agree, its kinda hacky. I would be happy to go with any alternatives if any.
using `kubectl config` there is only one field which could be renamed (`kubectl config rename-context`). for renaming cluster and user there are no commands or options.
I have updated the pr to put this logic within `if [[ -n "${KUBE_CONTEXT_NAME}" ]]` and also added TODO

We need to have merged kubeconfig for sure. sorry, if we have given other impressions.
the method i am considering to merge is as below:
```
# KUBECONFIG=us-central1-a/kubeconfig.json:us-central1-b/kubeconfig.json kubectl config view --raw >kubeconfig2.json
# kubectl --kubeconfig=kubeconfig.json config get-contexts
CURRENT   NAME            CLUSTER         AUTHINFO              NAMESPACE
*         us-central1-a   us-central1-a   us-central1-a-admin   
          us-central1-b   us-central1-b   us-central1-b-admin
```